lib/bitex_bot/robot.rb

```diff
@@ -14,10 +14,13 @@ module BitexBot
   #
   # rubocop:disable Metrics/ClassLength
   class Robot
-    cattr_accessor :graceful_shutdown
+    cattr_accessor(:orderbook) { Settings.bitex.orderbook }
```

Comment:
I'm not sure about these shortcuts, it does not make it obvious that it's referring to the Bitex orderbook.

Reply:
done